ANSWERS.md

```diff
@@ -0,0 +1,8 @@
+1. PropTypes are used for Typechecking. Typechecking is a way to catch erorrs and rule out errors that you can possibly get. This is important because as
+an application scales more bugs have the chance of being introduced and Typechecking can reduce the amount of possible errors.
+
```

Comment:
Yes that's true, however that answer is not complete. 

Specifically, the Typechecking that prop types does is data validation for properties being passed down to child components in a React application. They indicate to future developers the data that is required and intended to be passed into a component as props.